frontend/src/containers/RecipePage/Recipe.css

```diff
@@ -0,0 +1,154 @@
+.overall-container {
+  margin: 0px auto;
+  padding: 0px;
+}
+
+.user-edit-button {
+  display: inline-block;
+  float: right;
+  margin: 20px 26px;
+  border: 3px solid #ffa616;
```

Comment:
I think the border may need to be a bit thinner to match other buttons on the site/the design doc.

Reply:
changed button to match others